ui/src/components/workflow/errordashboard/ErrorDashboard.js

```diff
@@ -120,17 +125,26 @@ const ErrorDashboard = React.createClass({
            }
            else
            {
-             if(d.isRequiredInReporting == true){
+             if(d.isRequiredInReporting == true && includeAllErrors == false){
```

Comment:
Wont this mess up the reporting paging?
For ex..If the resultset has all ignorable errors, will this page show anything?